bika/lims/utils/__init__.py

```diff
@@ -890,6 +891,54 @@ def is_bika_installed():
     qi = api.portal.get_tool("portal_quickinstaller")
     return qi.isProductInstalled("bika.lims")
 
+def convert_unit(result, formula, precision):
+    """ take a value and a unit conversion formula and convert the result
+    """
+    try:
+        formula = formula.replace('Value', '%f')
+        new =  eval(formula % float(result))
+        fmt = '{{:.{}f}}'.format(precision)
```

Comment:
This format always uses `.` as decimal mark, which might not fit for all users. Better take the decimal mark as provided in the setup

Reply:
Done